env/base-sepolia.json

```diff
@@ -2,25 +2,20 @@
   "network": {
     "chainId": 84532,
     "environment": "testnet",
-    "centrifugeId": 2
+    "centrifugeId": 2,
+    "connectsTo": [
```

Comment:
For now it's just setting it up with the assumption that the adapters on chain A are always the same as the adapters on chain B, for every connection. That won't be true in reality, but let's consider this a next step.

Reply:
Agree, it is easier and clenaer. But will this hold true for now even in production at least for the time being?